packages/actions/tsconfig.eslint.json

```diff
@@ -1,3 +1,20 @@
 {
-	"extends": "../../tsconfig.eslint.json"
+	"extends": "./tsconfig.json",
+	"compilerOptions": {
+		"allowJs": true
+	},
```

Comment:
I don't think we should need this option for TypeScript-only packages, only `discord.js` would use this.